cfgov/ask_cfpb/models/blocks.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import absolute_import
+
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.forms.utils import ErrorList
+
+from wagtail.wagtailcore import blocks
+
+from ask_cfpb.models.django import Answer
+from v1 import blocks as v1_blocks
+from v1.atomic_elements import molecules
+
+
+class AskHeadingLevelBlock(blocks.ChoiceBlock):
+    choices = [
```

Comment:
I noticed that this is identical to the choices in `HeadingLevelBlock`, with one additional choice for bold.  Would it make sense to subclass that and call super, and then append the bold option to the list?  For me, it's not so much to reduce the duplicate lines, but would help with readability.